index.js

```diff
@@ -56,10 +61,19 @@ Converter.prototype.toJSON = function (space) {
   return JSON.stringify(this.sourcemap, null, space);
 };
 
-Converter.prototype.toBase64 = function () {
-  var json = this.toJSON();
-  return (SafeBuffer.Buffer.from(json, 'utf8') || "").toString('base64');
-};
+Converter.prototype.toBase64 = typeof Buffer !== 'undefined' ? Buffer.from ?
+  function () {
```

Comment:
Can these be `if` statements too?

Reply:
Agreed + this function should be declared elsewhere and named and then just referenced here, otherwise it is hard to debug since it'll be anonymous.

Additionally probably should do `Buffer.from.bind(Buffer` just in case in the future that function will refer to `Buffer` as `this`.